test/sharness/t0172-content-routing-over-http.sh

```diff
@@ -30,12 +30,16 @@ test_expect_success 'create unique CID without adding it to the local datastore'
   WANT_CID=$(date +"%FT%T.%N%z" | ipfs add -qn)
 '
 
-test_expect_success 'expect HTTP request for unknown CID' '
+test_expect_success 'expect HTTP lookup when CID is not in the local datastore' '
   ipfs block stat "$WANT_CID" &
   test_wait_output_n_lines_60_sec http_requests 3 &&
   test_should_contain "GET /routing/v1/providers/$WANT_CID" http_requests
 '
 
+test_expect_success 'expect HTTP request User-Agent to match Kubo version' '
```

Comment:
would be nice to add this using the new sharness go framework.

Reply:
Yes, extracted to https://github.com/ipfs/kubo/issues/9562  so test refactor does not block  0.18

Reply:
Actually, it is flaky enough, I'll do it here.